src/templates/meeting-room-schedule/meeting-room-schedule.js

```diff
@@ -0,0 +1,134 @@
+import { React, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import dayjs from 'dayjs';
+import { IntlProvider, FormattedMessage } from 'react-intl';
+import localeDa from 'dayjs/locale/da';
+import localizedFormat from 'dayjs/plugin/localizedFormat';
+import './meeting-room-schedule.scss';
+
+/**
+ * Meeting room schedule component.
+ *
+ * @param {object} props
+ *   Props.
+ * @param {object} props.content
+ *   The meeting room schedule content.
+ * @returns {JSX.Element}
+ *   The component.
+ */
+function MeetingRoomSchedule({ content }) {
+  dayjs.extend(localizedFormat);
```

Comment:
Maybe this should be placed in the top of `useEffect(() => {}, []);` to avoid being called at each render.